guides/common/modules/proc_importing-container-images.adoc

```diff
@@ -33,9 +30,18 @@ Click to edit, and add any tags that you want to limit the content that synchron
 . In the {ProjectWebUI}, navigate to *Content* > *Products* and select the name of your product.
 . Select the new repositories and then click *Sync Now* to start the synchronization process.
 
-To view the progress of the synchronization, navigate to *Content* > *Sync Status* and expand the repository tree.
+.Procedure with creating a repository manually
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+Click the name of the required product.
+. Click *New repository*.
+. From the *Type* list, select *docker*.
+Enter the details for the repository, and click *Save*.
```

Comment:
IMHO needs a list of steps what do do/fill in for each field.

Reply:
I don't think so. The fields are described in the UI form itself, I would just be copy-pasting the descriptions in the docs.

![new_repo_scrn](https://github.com/theforeman/foreman-documentation/assets/14296616/51278eaf-44e8-482f-96a2-0039c07c4b3f)

Reply:
That's what we currently do, don't we?

* https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Creating_a_Custom_Product_content-management
* https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Creating_a_Sync_Plan_content-management
* https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Creating_an_Activation_Key_content-management
* https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html#creating-operating-systems_provisioning
* etc

Reply:
We do, and it's not something I support. I believe it creates an undesirable user experience and is also a maintenance burden for writers.

We can discuss this in more detail at the next meeting if you want to. Or should we move this conversation to an issue?

Reply:
Yes, please open an issue for this.

Reply:
https://github.com/theforeman/foreman-documentation/issues/2490

I'm looking forward to the discussion!